data.js

```diff
@@ -208,6 +208,41 @@ exports.data = [
             link: "https://www.wri.org/",
             label: "world-resource-institution",
         },
+        established: {
+            content: "1982",
+            updatedBy: "Andre", updatedAt: "2020-01-04",
+        },
+        kpis: {
+            content: "- Organizations using their guidelines \n- Contracts signed \n- Articles wrote \n- Press mentions",
+            updatedBy: "Andre", updatedAt: "2020-02-13",
+
+        },
+        description: {
+            content: "Another big global player fighting climate crisis. Their approach is to measure their work. Their motto is 'count it, change it, scale it'.",
+            updatedBy: "Andre", updatedAt: "2020-01-13",
+        },
+        
+        actions: {
+            content: "Examples of [2018](https://www.wri.org/annualreport/2018-19#page=4):"+
+                     "\n- Guidlines of WRI were adopted by several cities to develope a more sustainable traffic."+ 
+                     "\n- Were lobbying for the Paris Agreement."+
+                     "\n- Helped to implement first binding environmental laws in 16 South American states."+
```

Comment:
South America has 13 countries. If this is referring to states it seems necessary to mention of which countries they are. It differs from country to country what a state means (e.g. France is not really federal)

Reply:
researched better